lib/vsc/utils/fancylogger.py

```diff
@@ -341,15 +343,22 @@ def getLogger(name=None, fname=False, clsname=False, fancyrecord=None):
     if fname is True, the loggers name will be 'name[.classname].functionname'
     if clsname is True the loggers name will be 'name.classname[.functionname]'
     This will return a logger with a fancylog record, which includes the className template for the logformat
-    This can make your code a lot slower, so this can be dissabled by setting fancyrecord to False, and
-    will also be disabled if a Name is set, and fancyrecord is not set to True
+    This can make your code a lot slower, so this can be dissabled by setting fancyrecord or class module
+    FANCYLOG_FANCYRECORD to False, or will also be disabled if a Name is set (and fancyrecord and
+    module constant FANCYLOG_FANCYRECORD are also not set).
     """
     nameparts = [getRootLoggerName()]
 
+    if fancyrecord is None:
+        # Altough we could set it as default value in the function definition
+        # it's easier to explain if we do it this way
+        fancyrecord = FANCYLOG_FANCYRECORD
+
     if name:
         nameparts.append(name)
-    elif fancyrecord is None or fancyrecord:  # only be fancy if fancyrecord is True or no name is given
+    elif fancyrecord is None:  # only be fancy if fancyrecord is True or no name is given
         fancyrecord = True
+
     fancyrecord = bool(fancyrecord)  # make sure fancyrecord is a nice bool, not None
```

Comment:
maybe add a test case that checks that things still work when passing a non-boolean value then?

```python
logger = fancylogger.getLogger('myname', fancyrecord='yes')
self.assertEqual(logger.fancyrecord, True)

logger = fancylogger.getLogger('myname', fancyrecord=0)
self.assertEqual(logger.fancyrecord, False)
```